hw_diag/tasks.py

```diff
@@ -1,30 +1,54 @@
 import logging
 import datetime
 import json
+from typing import Any
 
 from hm_pyhelper.hardware_definitions import variant_definitions
 from hm_pyhelper.miner_param import get_ethernet_addresses
-from hw_diag.utilities.blockchain import get_helium_blockchain_height
+from hw_diag.diagnostics.lte_diagnostic import LOGGER
 from hw_diag.utilities.hardware import detect_ecc
 from hw_diag.utilities.hardware import get_serial_number
 from hw_diag.utilities.hardware import lora_module_test
 from hw_diag.utilities.hardware import set_diagnostics_bt_lte
 from hw_diag.utilities.hardware import get_public_keys_and_ignore_errors
-from hw_diag.utilities.miner import fetch_miner_data
 from hw_diag.utilities.shell import get_environment_var
 from hw_diag.utilities.gcs_shipper import upload_diagnostics
-from hm_pyhelper.miner_json_rpc.exceptions import MinerFailedFetchData
-from requests.exceptions import ConnectTimeout, ReadTimeout
+from hw_diag.diagnostics.gateway_diagnostics import GatewayDiagnostic
+from hm_pyhelper.diagnostics import DiagnosticsReport
 
 
 log = logging.getLogger()
 log.setLevel(logging.DEBUG)
 
 
+def get_deep_key(source_dict: dict, key_path: list) -> Any:
```

Comment:
Can you point me to where this method is used? Can't find it.

Reply:
removed.